### PR TITLE
[JS] Upgrade a npm package version

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Related tickets | 
| License         | MIT

Proper version in package.json file

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "sylius",
-  "version": "0.17.0",
+  "version": "0.18.0",
```
Comment (pamil) [inline package.json]: Isn't it `0.19.0` on master right now? :)

**Comment by pjedrzejewski** [comment]
I am pretty sure we should remove this and all other configurations which may suggest that this is a standalone package on NPM. We should only keep our requirements and that's it, wdyt?

File: package.json (hunk 1 above)
Comment (lchrusciel) [inline package.json]: Right now we have `0.19.0-dev` so it is not a `0.19.0` directly

**Comment by lchrusciel** [comment]
@pjedrzejewski it makes sense. I have just found a forgotten piece of code. 👍 for removal.

**Comment by pjedrzejewski** [comment]
@lchrusciel Could you do it in this PR? Thanks!

**Comment by lchrusciel** [comment]
sure, no problem

**Comment by pjedrzejewski** [comment]
Thanks Łukasz!

**Pull request merged**